service/src/main/java/uk/gov/hmcts/reform/fpl/service/PbaNumberService.java

```diff
@@ -44,6 +46,13 @@ public C2DocumentBundle update(C2DocumentBundle c2DocumentBundle) {
             .orElse(c2DocumentBundle);
     }
 
+    public String update(String pbaNumber) {
```

Comment:
can be 
    public String update(String pbaNumber) {
        return ofNullable(pbaNumber)
            .map(PbaNumberHelper::setPrefix)
            .orElse(null);
    }

Reply:
nice thanks :)

Reply:
changing back as fails my test as it tries to set prefix when pba is empty